src/main/java/ru/urururu/sanity/cpp/Parser.java

```diff
@@ -48,10 +49,14 @@
     ConstCache constants;
 
     public List<Cfg> parse(String filename) throws ParseException {
+        System.out.println("filename = " + filename);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)